src/scala/com/github/johnynek/bazel_deps/S3URLConnection.scala

```diff
@@ -0,0 +1,102 @@
+package com.github.johnynek.bazel_deps
+
```

Comment:
is this file mostly original code, or largely copied? If copied can you provide some git permalinks here to where it came from to help others.

Also, is there any way we could use the compiled version of the original code rather than adding as much code here?

Reply:
Larged copied, I was doing this just to unblock me, so I didn't really think it through a lot. That being said, I'm really open to make any modifications to help others to use it.

Reply:
My first thinking of bringing the code here was to avoid any reference to `sbt` int the code. But maybe I could try to use as a dependency and just override what is needed by this project. I'll see if it works out, if not I'll add links to github and better documentation for this part of the code